data/virtualization/autoyast/guest_12.xml.ep

```diff
@@ -51,7 +51,7 @@
     <arch>{{ARCH}}</arch>
     <release_type/>
     </addon>
-    % if ($ltss_code) {
+    % if ($ltss_code && $check_var->('EXTENDED_SECURITY', '0') || $ltss_code && !$ltss_es_code && $check_var->('EXTENDED_SECURITY', '1')) {
```

Comment:
What to be checked here ?  ```A && B || C && !D && E```. This checking looks not scientific enough.

Reply:
A && B = sle12 guest with ltss keys will use ltss key to register, when we are not testing ltss_es.
C && !D && E = When testing ltss_es we will register all sle12 ltss guests that have ltss key, except the guest with ltss_se key

But on second look I think `C && !D && E ` is redundant. Only sles12 guest we test when `EXTENDED_SECURITY=1` is sles12sp5. I will remove `C && !D && E` .

Reply:
Make sure you implement when you want. Also add parentheses to make it better if necessary.